pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -183,50 +190,37 @@ async def run(self):
             jira_issue = self._jira_client.get_issue(jira_issue_key)
             subtasks = [self._jira_client.get_issue(subtask.key) for subtask in jira_issue.fields.subtasks]
             self.update_release_jira(jira_issue, subtasks, jira_template_vars)
-            _LOGGER.info("Updating prepare release subtask")
-            parent_jira = self._jira_client.get_issue(jira_issue_key)
-            subtask = self._jira_client.get_issue(parent_jira.fields.subtasks[1].key)
-            self._jira_client.add_comment(
-                subtask,
-                "prepare release job : {}".format(os.environ.get("BUILD_URL"))
-            )
-            self._jira_client.assign_to_me(subtask)
-            self._jira_client.close_task(subtask)
-            self._jira_client.start_task(parent_jira)
         else:
             _LOGGER.info("Creating a release JIRA...")
             jira_issues = self.create_release_jira(jira_template_vars)
             jira_issue = jira_issues[0] if jira_issues else None
             jira_issue_key = jira_issue.key if jira_issue else None
-            _LOGGER.info("Updating prepare release subtask")
-            if jira_issue_key:
-                parent_jira = self._jira_client.get_issue(jira_issue_key)
-                subtask = self._jira_client.get_issue(parent_jira.fields.subtasks[1].key)
+
+        if jira_issue_key:
+            _LOGGER.info("Updating Jira ticket status...")
+            if not self.runtime.dry_run:
+                subtask = jira_issue.fields.subtasks[1]
                 self._jira_client.add_comment(
                     subtask,
                     "prepare release job : {}".format(os.environ.get("BUILD_URL"))
                 )
                 self._jira_client.assign_to_me(subtask)
                 self._jira_client.close_task(subtask)
-                self._jira_client.start_task(parent_jira)
+                self._jira_client.start_task(jira_issue)
+            else:
+                _LOGGER.warning("[DRY RUN ]Would have updated Jira ticket status")
 
         _LOGGER.info("Updating ocp-build-data...")
         build_data_changed = await self.update_build_data(advisories, jira_issue_key)
 
         _LOGGER.info("Sweep builds into the the advisories...")
-        for kind, advisory in advisories.items():
+        for impetus, advisory in advisories.items():
```

Comment:
Personally I like the variable name `kind` better than `impetus` in this case, it makes it obvious what I'm looking at. I'm not sure what the intent behind chosing impetus was in our other commands

Reply:
We already use `kind` and `impetus` interchangeably in our code. It is confusing. Sometimes `kind` means `rpm` or `image`, but sometimes it means `rpm` `image` `extras` `metadata`. We also use `impetus` for the later.

I am not sure if everyone feel comfortable if we use `impetus` for `rpm` `image` `extras` `metadata`.